pom.xml

```diff
@@ -61,6 +61,18 @@
             <artifactId>oro</artifactId>
             <version>2.0.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.opencb.biodata</groupId>
```

Comment:
You're right, I tried compiling it without this and it worked fine. Removed in a subsequent commit.